scripts/core/ArticlesListByQueryWithFilters.tsx

```diff
@@ -350,7 +350,10 @@ export class ArticlesListByQueryWithFilters extends React.PureComponent<IProps,
                                         </h3>
                                         <Badge type="default">{itemsCount}</Badge>
                                     </div>
-                                    <div style={{marginLeft: 10, flexGrow: 1}}>
+                                    <div
+                                        className="sd-display--flex sd-flex--grow
```

Comment:
double quotes don't support being used across multiple lines.

instead use:
```
<div
    className={classnames(
        'class1',
        'class2',
        'class3',
    )}
/>
```

Reply:
Done.

Reply:
Good to know :)